File: Java-Core/src/main/java/presto/python/PythonLiteral.java
package presto.python;

import presto.utils.CodeWriter;

public abstract class PythonLiteral implements PythonExpression {
	
	String text;
	
	protected PythonLiteral(String text) {
		this.text = text;
	}
	
	@Override
	public void toDialect(CodeWriter writer) {
		writer.append(text);
	}
}

File: Java-CodeServer/src/main/resources/js/prompto/python/PythonLiteral.js
function PythonLiteral(text) {
    this.text = text;
    return this;
}

PythonLiteral.prototype.toString = function() {
    return this.text;
};

PythonLiteral.prototype.toDialect = function(writer) {
    writer.append(this.text);
};

exports.PythonLiteral = PythonLiteral;